### ISSUE TITLE
Mask Detection using EfficientnetB3

### ISSUE THREAD

**Issue by NEERAJAP2001** [CoinShift-Imaging/CoinShift-Imaging-Box#77]
On an additional note, I'd like to add a Live Webcam Feature to detect masks using EfficientNetB3.

# Architecture : 

```
tf.keras.applications.EfficientNetB3(
    include_top=True, weights='imagenet', input_tensor=None,
    input_shape=None, pooling=None, classes=1000,
    classifier_activation='softmax', **kwargs
)

```

# Project Pipeline : 

![face mask detection- mtcnn](https://user-images.githubusercontent.com/65017645/114337677-305cdf00-9b6f-11eb-8e22-f53bc57e3b7a.png)

I'd like to work on this issue as part of GSSOC-21 and this will be implemented into [This](https://github.com/CoinShift-Imaging/CoinShift-Imaging-Box/tree/main/Face%20Mask%20Detection) Folder.

**Comment by DARK-art108** [comment]
You can work upon it Go Ahead :)
